### PR TITLE
Leaderboard setup

### PR DESCRIPTION
**Pull request by Ukaypromise**
## Setting up Leaderboard Wireframe

1. Create new branch
2. Add webpack setup.
3. Add linter setup.
4. Fix linter errors.
5. Update README.md file.
6. Introduced modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (zieeco) [inline README.md]: - Please, kidly add the missing license content as the link is broken to make your readme more professional.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,63 @@
...
+## Authors
+👤 **Promise Uka**
+
+- GitHub: [@Ukaypromise](https://github.com/Ukaypromise/)
+- Twitter: [@PromiseUkay](https://twitter.com/PromiseUkay)
+- LinkedIn: [@promiseuka](https://www.linkedin.com/in/promiseuka)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```
Comment (zieeco) [inline README.md]: - Please, kindly ensure you've added the issues link to your project Github repository to make it easy for users to navigate to the issues page which would improve the user experience and make your readme more professional.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,63 @@
...
+npm install
+`
+### Usage
+`
+npx hint .
+npx stylelint "**/*.{css,scss}"
+npx eslint .
+`
+
+## Authors
```
Comment (zieeco) [inline README.md]: - Please, kindly make the authors singular as you've worked on this project alone.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
 # Leaderboard
-Leaderboard list app, using webpack and ES6 modules.
+Leaderboard list app, using webpack and ES6 modules.  The projeect consume the Leaderboard API using JavaScript async and await methods.
+
+![Leaderboard](../leader.png)
```
Comment (zieeco) [inline README.md]: - Please, kindly fix the issue with the broken link of your screenshot image. You can update it or remove it as it won't be ideal leaving a broken image link in your readme file.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Ukaypromise,


Great so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**